models/noah/README.rst

```diff
@@ -0,0 +1,35 @@
+###########
+Noah README
+###########
```

Comment:
I would like the `wrf_hydro/README.rst` duplicated in the `noah/README.rst`.  As it stands, the `wrf_hydro/README.rst` seems like it actually has the (lanai) version of the documentation for NOAH.